src/main/java/hw6_jdi/jdiPages/JdiMetalsColorsPage.java

```diff
@@ -0,0 +1,44 @@
+package hw6_jdi.jdiPages;
+
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import hw6_jdi.entities.MetalsColorsResult;
+import hw6_jdi.forms.MetalsColorsForm;
+
+public class JdiMetalsColorsPage extends WebPage {
+    @Css("div.info-panel-section ul.panel-body-list.results li")
+    public WebList resultBlock;
+
+    public HeaderSection headerSection;
+    public LeftMenuSection leftMenuSection;
+    public MetalsColorsForm metalsColorsForm;
+
+    public MetalsColorsResult convertResultBlockToClass() {
+        MetalsColorsResult metalsColorsResult = new MetalsColorsResult();
```

Comment:
Basically, you don't really need to check the results with reference to field's order. Therefore, it will be quite easier to compare two lists of strings. Ferst list contains with data from the entity and the second one from the web element from the page.